### ISSUE TITLE
Legibility problems on Steam Deck

### ISSUE THREAD

**Issue by BenLubar** [ReactiveDrop/reactivedrop_public_src#763]
While getting re-verified to move to a newer version of Proton for the font rendering fix, some issues popped up that will make us temporarily lose our Steam Deck Verified status. One of them is a technical issue that will be fixed in the next beta, one is a legibility issue which is what this ticket is for, and one is a technical issue that causes the game to have an Unsupported status that Valve is looking into from their end and I will also look into from my end after posting this.

Here are the screenshots shared with me by the member of Valve who reviewed the game's compatibility with Steam Deck (redactions are mine; just hiding display name and avatar).

![Alien Swarm Reactive Drop - Legibility 1](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/d9adf3ee-f68b-4dd9-adeb-63a076ba0689)
![Alien Swarm Reactive Drop - Legibility 2](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/8b9d43f3-a928-4893-a4df-7008e165f6dd)
![Alien Swarm Reactive Drop - Legibility 3](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/a795760d-eb95-46f8-bf62-0c864de277ef)
![Alien Swarm Reactive Drop - Legibility 4](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/132ba66a-74c4-471c-bce3-3a64d8032b32)
![Alien Swarm Reactive Drop - Legibility 5](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/5e945529-163f-46b6-a7ad-9809432ec8e2)
![Alien Swarm Reactive Drop - Legibility 6](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/6cc28288-df54-419b-b383-da5d0df9e492)
![Alien Swarm Reactive Drop - Legibility 7](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/38fd635a-9ac2-481e-b222-01f5955f3237)
![Alien Swarm Reactive Drop - Legibility 8](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/175ba286-3479-429a-ab5e-ab1c9bdea938)
![Alien Swarm Reactive Drop - Legibility 9](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/47e47638-5e42-4e07-ab5f-3c754ef04115)
![Alien Swarm Reactive Drop - Legibility 10](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/9c4157a3-07d2-4acd-9709-5ec8d1c1f37f)
![Alien Swarm Reactive Drop - Legibility 11](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/aba6cfbc-45f9-47dd-8a1d-e4552d6d2249)
![Alien Swarm Reactive Drop - Legibility 12](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/543d24ca-e486-4733-a507-6bf4205b6e42)
![Alien Swarm Reactive Drop - Legibility 13](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/e9d28cc1-b3f6-449f-bd9e-195335c60206)
![Alien Swarm Reactive Drop - Legibility 14](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/d6081ae9-d724-4207-b30d-98ddb523a646)
![Alien Swarm Reactive Drop - Legibility 15](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/6075e0e4-2b48-43bb-8c0d-802b044d2afa)
![Alien Swarm Reactive Drop - Legibility 16](https://github.com/ReactiveDrop/reactivedrop_public_src/assets/4257305/68d90fd2-c44f-4ec8-9d38-440541765e7f)

Basically every screen in the game has text that's too small, and the smallest font sizes are rendering as the wrong font due to Verdana not being available in the default Proton environment.

I don't think we can legally distribute Verdana with the game, so what we need to do is:

- [ ] Change any font that assumes a Windows system font will be available, unless Proton has those fonts available by default.
- [ ] Make a new version of the game's UI optimized for Big Picture mode. We can probably route the settings screen to the existing L4D2 style UI, but pretty much everything else needs to be adjusted if not completely split and redone.
